Source/non_ip/ai.conf

```diff
@@ -53,3 +53,10 @@ DOMAIN-SUFFIX,clipdrop.co
 DOMAIN-SUFFIX,jasper.ai
 # OpenArt
 DOMAIN-SUFFIX,openart.ai
+# GitHub Copilot
+DOMAIN-SUFFIX,githubcopilot.com
+DOMAIN,copilot-proxy.githubusercontent.com
```

Comment:
Did you find any references about region restrictions on those domains?

Reply:
Here is my finding after verification:

After packet analysis, enabling the 'Claude features' flag only requires adding two additional rules:

```
DOMAIN-SUFFIX,githubcopilot.com  # Required for IP & ASN reporting to obtain regions flag
DOMAIN,api.github.com            # Required for IP & ASN retrieval
```

Sample request 1:
```http
Host: api.github.com
Path: /copilot_internal/v2/token
Authorization: [redacted]
```
Body:
```json
{
  "token": "tid=;exp=;sku=;proxy-ep=proxy.individual.githubcopilot.com;...;ip=0.0.0.0;asn=AS114514:..."
}
```

Sample request 2:
```http
Host: api.individual.githubcopilot.com
Path: /models
Authorization: ...[Sample1][token]
```
I apologize for my previous incorrect conclusion. The above findings have been verified through network traffic analysis.

Reply:
It seems that `api.github.com` is receiving the information about IP & ASN from the POST body, not reading the IP on the server-side. So have you tried only adding `githubcopilot.com` without `api.github.com`?

Reply:
It seems that `api.individual.githubcopilot.com` is receiving the token information in which the IP & ASN information is included.
So have you tried only adding `api.github.com` only w/o `api.individual.githubcopilot.com`? I'd like to know if GitHub is enforcing same IP & ASN between the `api.github.com` and `api.individual.githubcopilot.com` domains.

Reply:
Very sharp observation, as I reconfigured the rules and tested them I found that api.github.com was the deciding factor, without including githubcopilot.com 👍